planner/core/common_plans.go

```diff
@@ -263,6 +264,12 @@ func (e *Execute) checkPreparedPriv(ctx context.Context, sctx sessionctx.Context
 	return err
 }
 
+func (e *Execute) addHitInfo(sctx sessionctx.Context, opt string) error {
```

Comment:
The function name is `addHitInfo`, but it actually manipulates the variable `TiDBFoundInPlanCache`. Please rename it to another proper name.

BTW, it's better to change the `opt` to boolean type. Thus we don't need to pass magic string like "1" or "0" to this function.